modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -28,11 +41,60 @@ export interface UploadResult {
 }
 
 interface EditorUpload {
+  /**
+   * Cache of blob elements created in an editor instance.
+   *
+   * @property blobCache
+   * @type Object
+   */
   blobCache: BlobCache;
+
+  /**
+   * Adds a custom filter that controls the images which are included in the scan.
+   * Images must return true on every added filter to be considered valid.
+   *
+   * @method addFilter
+   * @param {Function} filter Function which filters each image upload.
+   * @example
+   * // Filter which images are uploaded.
+   * tinymce.activeEditor.EditorUpload.addFilter((image) => {
+   *   const maxSize = 1920 * 1080;
+   *   const imageSize = image.width * image.height;
+   *   return imageSize < maxSize;
+   * });
+   */
   addFilter: (filter: (img: HTMLImageElement) => boolean) => void;
+
+  /**
+   * Uploads all the data uri/blob uri images scanned from the editor content to the server.
+   *
+   * @method uploadImages
+   * @return {Promise} Promise instance with images and status for each image.
+   */
   uploadImages: () => Promise<UploadResult[]>;
+
+  /**
+   * Uploads all data uri/blob uri images to the server only when automatic uploads are enabled.
+   *
+   * @method uploadImagesAuto
+   * @return {Promise} Promise instance with images and status for each image.
+   */
   uploadImagesAuto: () => Promise<UploadResult[]>;
+
+  /**
+   * Scans the editor content for valid image elements and generates blob information for each image.
+   *
+   * @method scanForImages
+   * @return {Promise} Promise instance with element object and blob information for each image.
+   */
   scanForImages: () => Promise<BlobInfoImagePair[]>;
+
+  /**
+   * Resets the blob cache, uploads and image scanner. Called automatically on <code>editor.remove</code>.
+   * This method is not recommended to be used by integrators.
```

Comment:
```suggestion
   * Resets uploads, the image scanner, and the blob cache. Called automatically on <code>editor.remove</code>.
   * This method is not recommended for integrator use.
```
1. Changes the presentation order of items this method resets. To my ear, placing the nouns in definite form after the noun in indefinite form sounds better. (Also, _image scanner_ reads as absolutely needing to be in the definite form.)
2. Switching in a preposition for the infinitive present tense verb reads less awkwardly, at least to me.

Reply:
Should make more sense now, higher level description.